test/sampledata/dataspace/MasterDataspace/study/datasets/datasets_metadata.xml

```diff
@@ -794,19 +795,49 @@
         <columnTitle>Titer IC50</columnTitle>
         <description>The half maximal inhibitory concentration (IC50).</description>
         <rangeURI>http://www.w3.org/2001/XMLSchema#double</rangeURI>
+        <isHidden>true</isHidden>
       </column>
       <column columnName="titer_ic80">
         <datatype>integer</datatype>
         <columnTitle>Titer IC80</columnTitle>
         <description>The 80% maximal inhibitory concentration (IC80).</description>
         <rangeURI>http://www.w3.org/2001/XMLSchema#double</rangeURI>
+        <isHidden>true</isHidden>
+      </column>
+      <column columnName="titer_ID50">
+        <datatype>double</datatype>
+        <columnTitle>Titer ID50</columnTitle>
+        <description>The adjusted value of 50% maximal inhibitory dilution (ID50).</description>
+        <rangeURI>http://www.w3.org/2001/XMLSchema#double</rangeURI>
+      </column>
+      <column columnName="titer_ID80">
+        <datatype>double</datatype>
+        <columnTitle>Titer ID80</columnTitle>
+        <description>The adjusted value of 80% maximal inhibitory dilution (ID80).</description>
+        <rangeURI>http://www.w3.org/2001/XMLSchema#double</rangeURI>
+      </column>
+      <!-- Could this be a boolean? Spec says value is 1 or 0 -->
+      <column columnName="nab_response_ID50">
+        <datatype>integer</datatype>
+        <columnTitle>Response call ID50</columnTitle>
+        <description>Indicates if neutralization is detected based on ID50 titer.</description>
+        <rangeURI>http://www.w3.org/2001/XMLSchema#int</rangeURI>
+        <dimension>true</dimension>
+      </column>
+      <!-- Could this be a boolean? Spec says value is 1 or 0 -->
+      <column columnName="nab_response_ID80">
+        <datatype>integer</datatype>
+        <columnTitle>Response call ID80</columnTitle>
+        <description>Indicates if neutralization is detected based on ID80 titer.</description>
+        <rangeURI>http://www.w3.org/2001/XMLSchema#int</rangeURI>
+        <dimension>true</dimension>
       </column>
       <column columnName="response_call">
         <datatype>boolean</datatype>
         <columnTitle>Response call</columnTitle>
         <description>Indicates if neutralization is detected.</description>
         <rangeURI>http://www.w3.org/2001/XMLSchema#boolean</rangeURI>
-        <dimension>true</dimension>
+        <isHidden>true</isHidden>
```

Comment:
Though hidden, response call is still a dimension. Also, for cds, we typically hide variable in measure.js. <isHidden> will still be respected, but probably shouldn't use it here to be consistent with how we handle other variables.